invokeai/app/invocations/latent.py

```diff
@@ -198,7 +199,7 @@ def get_conditioning_data(
             # for ddim scheduler
             eta=0.0,  # ddim_eta
             # for ancestral and sde schedulers
-            generator=torch.Generator(device=unet.device).manual_seed(0),
+            generator=torch.Generator(device=unet.device).manual_seed(get_random_seed()),
```

Comment:
If you're not using a specific seed, I don't think you need to `manual_seed` at all. But I guess it doesn't hurt.

Reply:
Appears that the initial seed used is the same every time:
```py
>>> import torch
>>> torch.Generator().initial_seed()
67280421310721
# restart the repl just to be extra sure its fresh
>>> import torch
>>> torch.Generator().initial_seed()
67280421310721
```

But I did notice this function in the docs, which appears to be the blessed way to seed the generator:
```py
>>> g = torch.Generator()
>>> g.initial_seed()
67280421310721
>>> g.seed()
15390967034040818360
>>> g.initial_seed()
15390967034040818360
>>> g.seed()
11307471257349742134
>>> g.seed()
17364437299175068371
>>> g.seed()
5841587831953547992
```

I'm changing the PR to use that.